src/main.rs

```diff
@@ -19,40 +20,55 @@ async fn main() -> Result<()> {
         .expect("Failed to install crypto provider");
 
     let matches = cli::build_cli().get_matches();
-    let config = policy_server::config::Config::from_args(&matches)?;
+    match matches.subcommand_name() {
```

Comment:
I would prefer to avoid the whole indentation caused by this change.

I think doing something like that would leave the code more readable:

```
let matches = cli::build_cli().get_matches();
if matches.subcommand_name() == Some("docs") {
   // maybe use a different name for that function, that's the 1st thing that came to my mind
   return handle_docs_subcommand(&matches);
}

// leave the code unchanged